<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>497</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Simple Terminal</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>622</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionExit"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuPlot">
    <property name="title">
     <string>Plot</string>
    </property>
    <addaction name="actionAuto_scale_on"/>
    <addaction name="actionZoom"/>
    <addaction name="actionZoom_2"/>
    <addaction name="actionSave_Polt"/>
    <addaction name="actionSet_Color"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuPlot"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionAuto_scale_on">
   <property name="text">
    <string>Auto scale on</string>
   </property>
  </action>
  <action name="actionZoom">
   <property name="text">
    <string>Zoom +</string>
   </property>
  </action>
  <action name="actionZoom_2">
   <property name="text">
    <string>Zoom -</string>
   </property>
  </action>
  <action name="actionSave_Polt">
   <property name="text">
    <string>Save plot</string>
   </property>
  </action>
  <action name="actionSet_Color">
   <property name="text">
    <string>Set Color</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
